content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,101 @@ description: Upgrading an ambient mode installation with Helm.
-{{< /warning >}}
+{{< text syntax=bash snip_id=upgrade_istiod >}}
+$ helm install istiod-"$REVISION" istio/istiod -n istio-system --set revision="$REVISION"
+{{< /text >}}
+
+## Upgrade the ztunnel DaemonSet
 
-The ztunnel at version 1.x is generally compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
+The ztunnel DaemonSet is the node proxy component. The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
 
 {{< warning >}}
 Upgrading ztunnel in-place will briefly disrupt all ambient mesh traffic on the node.
 Node cordoning and blue/green node pools are recommended to mitigate blast radius risk during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
 {{< text syntax=bash snip_id=upgrade_ztunnel >}}
-$ helm upgrade ztunnel istio/ztunnel -n istio-system
+$ helm upgrade ztunnel istio/ztunnel -n istio-system --set revision="$REVISION"
 {{< /text >}}
 
-### Upgrade the CNI component
+## Upgrade the CNI DaemonSet
 
 The Istio CNI agent is responsible for detecting pods added to the ambient mesh, informing ztunnel that proxy ports should be established within added pods, and configuring traffic redirection within the pod network namespace. It is not part of the data plane or control plane.
 
-{{< warning >}}
-As ambient mode is not yet Stable, the following statement is not a compatibility guarantee, and is subject to change, or removal. Prior to reaching Stable status, this component and/or the control plane may receive breaking changes that prevent compatibility between minor versions.
-{{< /warning >}}
-
-The CNI at version 1.x is generally compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before Istio CNI, as long as their version difference is within one minor version.
+The CNI at version 1.x is  compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before Istio CNI, as long as their version difference is within one minor version.
 
 {{< warning >}}
 Upgrading the Istio CNI agent to a compatible version in-place will not disrupt networking for running pods already successfully added to an ambient mesh, but no ambient-captured pods will be successfully scheduled (or rescheduled) on the node until the upgrade is complete and the upgraded Istio CNI agent on the node passes readiness checks. If this is a significant disruption concern, or stricter blast radius controls are desired for CNI upgrades, node taints and/or node cordons are recommended.
```

Comment:
is this true? I think they will schedule and not have ambient capabilities @bleggett ?(unless you have taints, which is not documented here)

Reply:
Does the CNI interact with the control plane?  isn't it's job just redirecting traffic to the ztunnel?

Reply:
Yeah and I think I wrote this, it is slightly wrong - it should read:

"Upgrading the Istio CNI agent to a compatible version in-place will not disrupt networking for running pods already successfully added to an ambient mesh, but any new pods scheduled on the node _during_ the `istio-cni` upgrade process may not be properly captured and could leak traffic during the upgrade process, until the upgrade is complete and the upgraded Istio CNI agent on the node passes readiness checks. If this is a significant disruption concern, or stricter blast radius controls are desired for CNI upgrades, node taints and/or node cordons are recommended."

Reply:
My comment about 'not true' was about "but no ambient-captured pods will be successfully scheduled" btw

To your question - 
It **indirectly** interacts with the control plane because the things it does needs to be compatible with how the proxies are configured. it is not directly communicating though

Reply:
It indirectly interacts with ztunnel (dataplane) , which directly interacts with the controlplane, yep